src/components/Callout/styles.tsx

```diff
@@ -18,7 +18,7 @@ export const CalloutOuter = styled.div<CalloutProps>((props) => {
     margin-bottom: 0;
     margin-top: 1rem;
 
-    &:first-child {
+    &:first-of-type {
```

Comment:
These two selectors will often have the same effect but not always.  Just for my own understanding, why the switch to first-of-type?

Reply:
We were getting a warning, but it's not related to the upgrade, so I've removed these and they can be addressed at another time.